pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -214,6 +232,36 @@ func (m *manager) Run(ctx context.Context) {
 	}()
 }
 
+// Gernerate pod resize completed event message
+func (m *manager) podResizeCompletionMsg(allocatedPod *v1.Pod) string {
+	podResizeSource := &podResourceSummary{}
+	podutil.VisitContainers(&allocatedPod.Spec, podutil.InitContainers|podutil.Containers,
+		func(allocatedContainer *v1.Container, containerType podutil.ContainerType) bool {
+			allocatedResources, exists := m.GetContainerResourceAllocation(allocatedPod.UID, allocatedContainer.Name)
+			if exists {
+				allocation := containerAllocation{
+					Name:      allocatedContainer.Name,
+					Resources: allocatedResources,
```

Comment:
per Tim's suggestion (https://github.com/kubernetes/kubernetes/pull/130387#discussion_r2214443343):

```suggestion
					Resources: allocatedContainer.Resources,
```

You already have the allocated container, so no need to call `m.GetContainerResourceAllocation` to fetch `allocatedResouces`.